.travis.yml

```diff
@@ -6,15 +6,25 @@ rvm:
   - 2.4
   - 2.5
   - 2.6
+env:
+  matrix:
+    - GROONGA_MASTER=no GROONGA_LATEST=yes
+    - GROONGA_MASTER=yes GROONGA_LATEST=no
+    - GROONGA_MASTER=no GROONGA_LATEST=no
 # env:
 #   - GROONGA_MASTER=yes
 # matrix:
 #   allow_failures:
 #     - rvm: ruby-head
 before_install:
-  - curl --silent --location https://github.com/groonga/groonga/raw/master/data/travis/setup.sh | sh
   - |
-    if [ "${GROONGA_MASTER}" != "yes" ]; then
-      sudo apt-get -qq -y install groonga-tokenizer-mecab
+    if [ "${GROONGA_LATEST}" != "yes" ]; then
+      curl --silent --location https://github.com/groonga/groonga/raw/master/data/travis/setup.sh | sh
+    - |
```

Comment:
Is this `- | `  needed? (maybe I'm not familiar with it, so just ignore it if it is expected)